modules/photo/src/color_transfer.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include "math.h"
+#include <vector>
+#include "opencv2/photo.hpp"
+
+using namespace std;
+using namespace cv;
+
+void cv::colorTransfer(InputArray _src, InputArray _dst, OutputArray _color_transfer, int flag)
+{
+    Mat source = _src.getMat();
+    Mat target = _dst.getMat();
+    _color_transfer.create(source.size(), CV_8UC3);
+    Mat output = _color_transfer.getMat();
+
+    Mat img1 = Mat(source.size(),CV_32FC3);
+    source.convertTo(img1,CV_32FC3,1.0/255.0);
+
+    Mat img2 = Mat(target.size(),CV_32FC3);
+    target.convertTo(img2,CV_32FC3,1.0/255.0);
+
+    if(flag == 0)
+    {
+        cvtColor(img1,img1,COLOR_BGR2Lab);
```

Comment:
note that this is not the same as the L alpha beta color space used by reinhardt